### PR TITLE
Add style to contact section

### PR DESCRIPTION
**Pull request by haoyxc**
Add some styling to contact section and make the links look more like buttons. The color of the "button" links change when the user hovers over it

How the contact section looks:
image
When mouse hovers over:
image

### CODE REVIEW

**Review by fenghaolw — COMMENTED** [review]
It is a little difficult to review since the files changed include changes in #5 and #6. 

I wonder if there is a way to show diffbase or something like that

**Comment by haoyxc** [comment]
I just pulled master, so it should at least include changes from #5. I can pull #6 once I merge it!

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -51,6 +51,7 @@
   position: fixed;
   top: 0; 
   width: 100%; 
+  z-index: 10;
```
Comment (fenghaolw) [inline portfolio/src/main/webapp/style.css:54]: Why is this needed? :) I personally hate z-index:
https://www.reddit.com/r/ProgrammerHumor/comments/58zwnu/the_only_way_to_deal_with_zindex/

Is there a better way to achieve the "ordering" without using z-index?

Comment (haoyxc) [reply portfolio/src/main/webapp/style.css:54]: omg haha

Comment (haoyxc) [reply portfolio/src/main/webapp/style.css:54]: that was just so the header would be on top of everything else (and when you scroll the content of the page is below the header) -- but I can look into other ways to do that

Comment (haoyxc) [reply portfolio/src/main/webapp/style.css:54]: I tried making the display block (of both the header and the content) and added extra padding to the content. In this case, the header was beneath the content of the page. 
image

I also tried using position: relative on the divs but ran into the same issue
image

I also tried making the position of the header static (but it scrolls with the page)
(The header is no longer there)
image

This is what I wanted: (when you scroll, the header stays and the content is hidden beneath)
image

Do you know any way I can work around using z-index? Thank you!

Comment (fenghaolw) [reply portfolio/src/main/webapp/style.css:54]: Does "position: sticky" work?

If not, feel free to use "position: fix" and z-index.

Comment (haoyxc) [reply portfolio/src/main/webapp/style.css:54]: image
ah sticky gives me the same issue where the content as above. 

"position: sticky" with the z-index works though (not sure if this is any better than "position: fixed" though)

Comment (fenghaolw) [reply portfolio/src/main/webapp/style.css:54]: It is probably not, so either way works for me :)

CSS is hard

**Review by fenghaolw — APPROVED** [review]
